.travis.yml

```diff
@@ -9,7 +9,7 @@ before_script:
   - ./cc-test-reporter before-build
 
 script:
-  - go test -gcflags=-l -coverprofile c.out ./...
+  - go test -gcflags=-l -run W -coverprofile c.out ./...
```

Comment:
the test must run sequentially
https://stackoverflow.com/questions/31201858/how-to-run-golang-tests-sequentially?answertab=votes#answer-31204016
because of monkey patch `os.Exit()` function